java/arcs/core/host/EntityHandleManager.kt

```diff
@@ -332,20 +379,38 @@ internal class WritableCollectionHandleImpl<T : Entity>(
 
 internal class ReadWriteCollectionHandleImpl<T : Entity>(
     entitySpec: EntitySpec<T>,
-    handleName: String,
+    val handleName: String,
```

Comment:
If best practice is `val`, should all of these ctor params be `val`? (I am the dog with chemistry beaker here, but asking while reading). @SHeimlich

Reply:
Do we need `val handleName` at all? Could these parameters just be `override val name`, since that property is already defined in a base class?